static/js/modules/helpers.js

```diff
@@ -118,6 +146,28 @@ function buildUrl(path, query) {
     .toString();
 }
 
+function getTimePeriod(electionYear, cycle, electionFull, office) {
+  var durations = {
+    P: 3,
+    S: 5,
+    H: 1
```

Comment:
These offsets seem a bit odd to me.  For instance, if I'm looking at a presidential election candidate, I'm seeing a full cycle of 2013 - 2016 being used.  Is this intended?  I am probably just misunderstanding the reasoning here, but wanted to call it out just in case.